search/src/main/java/artic/edu/search/DefaultSearchSuggestionsFragment.kt

```diff
@@ -69,10 +71,11 @@ class DefaultSearchSuggestionsFragment : BaseViewModelFragment<DefaultSearchSugg
                     when (navigation.endpoint) {
                         is DefaultSearchSuggestionsViewModel.NavigationEndpoint.ArticObjectDetails -> {
                             val o = (navigation.endpoint as DefaultSearchSuggestionsViewModel.NavigationEndpoint.ArticObjectDetails).articObject
-                            Navigation.findNavController(requireActivity(), R.id.container).navigate(
-                                    R.id.goToSearchAudioDetails,
-                                    SearchAudioDetailFragment.argsBundle(o)
-                            )
+                            val mapIntent = NavigationConstants.MAP.asDeepLinkIntent().apply {
+                                putExtra("ARG_SEARCH_OBJECT", o)
```

Comment:
.....Wonder if that string shouldn't be in `NavigationConstants` too.

Reply:
That sounds right. Moved to `NavigationConstants`